yarn.lock

```diff
@@ -6101,10 +6101,10 @@ levenary@^1.1.1:
   dependencies:
     leven "^3.1.0"
 
-libra-docusaurus@^0.4.10:
-  version "0.4.10"
-  resolved "https://registry.yarnpkg.com/libra-docusaurus/-/libra-docusaurus-0.4.10.tgz#36cf1bc250ce5a13bedec6665b202200241f9884"
-  integrity sha512-ysQFVF1DIKIwFYXy2YnpAs+bISNzGQo2zEQIej0Y6SWp6NzQSRKemjLNiYg2qhjilZAXtm1chpF9zYdnZnUfyA==
+libra-docusaurus@^0.5.1:
```

Comment:
When are you planning to upgrade the main Docusaurus version?

Reply:
This is just a poorly named library. It's actually a collection of shared components, not the main docusaurus library. This will be renamed to `libra-components` in the next PR.